decidim-conferences/app/commands/decidim/conferences/admin/notify_role_assigned_conference.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    module Admin
+    # A command to notify users when a role is assigned for a Conference
+      class NotifyRoleAssignedConference < Rectify::Command
+        def send_notification user
+            Decidim::EventsManager.publish(
```

Comment:
Something funny with the identation here!

Reply:
You can automatically run `bundle exec rubocop -a <list of files you want to check>`, or manually check the "Lint" job in the Checks section of this PR, it will tell you what's missing 😄